met-api/tests/unit/models/test_engagement.py

```diff
@@ -46,27 +45,43 @@ def test_get_engagements_paginated(session):
         size=None,
         sort_key='name',
         sort_order=''
-
     )
 
     # verify name search
     result, count = EngagementModel.get_engagements_paginated(pagination_options, eng.name)
     assert eng.name == result[0].name
     assert count == 1, 'Name search brings up only search result'
 
+
+def test_get_engagements_paginated_status_search(session):
+    """Assert that an engagement can be created and fetched."""
+    eng = factory_engagement_model()
+    for i in range(0, 10):
+        factory_engagement_model()
+    assert eng.id is not None
+    pagination_options = PaginationOptions(
+        page=None,
+        size=None,
+        sort_key='name',
+        sort_order=''
+    )
+
     # status search
     factory_engagement_model(status=SubmissionStatus.Closed.value)
     factory_engagement_model(status=SubmissionStatus.Closed.value)
-    result, count = EngagementModel.get_engagements_paginated(pagination_options, None, [SubmissionStatus.Closed.value])
+    result, count = EngagementModel.get_engagements_paginated(pagination_options, search_text=None,
+                                                              statuses=[SubmissionStatus.Closed.value])
     assert count == 2
 
-    result, count = EngagementModel.get_engagements_paginated(pagination_options, None, [SubmissionStatus.Open.value])
+    result, count = EngagementModel.get_engagements_paginated(pagination_options, search_text=None,
+                                                              statuses=[SubmissionStatus.Open.value])
     # 11 Open ones are created
     assert count == 11
     assert len(result) == 11
 
-    result, count = EngagementModel.get_engagements_paginated(pagination_options, None, [SubmissionStatus.Open.value,
-                                                                                         SubmissionStatus.Closed.value])
+    result, count = EngagementModel.get_engagements_paginated(pagination_options, search_text=None,
```

Comment:
just need one more with statuses=None or not passed (since it's optional parameter)